chef_master/source/release_notes.rst

```diff
@@ -2867,7 +2867,7 @@ The new ``gateway_identity_file`` option allows the operator to specify the key
 Windows Task resource added
 -----------------------------------------------------
 The ``windows_task`` resource has been ported from the windows cookbook.
-Use the **windows_task** resource to create, delete or run a Windows scheduled task. Requires Windows Server 2008 due to API usage.
```

Comment:
This should stay since this is a release note. We shouldn't 1984 rewrite history here.

Reply:
So, `Use the **windows_task** resource to create, delete or run a Windows scheduled task.` ?

Reply:
yep